docs/patterns/compute-resource-consolidation.md

```diff
@@ -79,159 +79,14 @@ Use this pattern for tasks that are not cost effective if they run in their own
-  {
-    Trace.TraceError(ex.Message);
-
-    // If any of the inner exceptions in the aggregate exception
-    // are not cancellation exceptions then re-throw the exception.
-    ex.Handle(innerEx => (innerEx is OperationCanceledException));
-  }
-
-  // If there wasn't a cancellation request, stop all tasks and return from Run()
-  // An alternative to canceling and returning when a task exits would be to
-  // restart the task.
-  if (!cts.IsCancellationRequested)
-  {
-    Trace.TraceInformation("Task returned without cancellation request");
-    Stop(TimeSpan.FromMinutes(5));
-  }
-}
-...
-```
-
-In this example, the `Run` method waits for tasks to be completed. If a task is canceled, the `Run` method assumes that the role is being shut down and waits for the remaining tasks to be canceled before finishing (it waits for a maximum of five minutes before terminating). If a task fails due to an expected exception, the `Run` method cancels the task.
-
-> You could implement more comprehensive monitoring and exception handling strategies in the `Run` method such as restarting tasks that have failed, or including code that enables the role to stop and start individual tasks.
-
-The `Stop` method shown in the following code is called when the fabric controller shuts down the role instance (it's invoked from the `OnStop` method). The code stops each task gracefully by canceling it. If any task takes more than five minutes to complete, the cancellation processing in the `Stop` method ceases waiting and the role is terminated.
-
-```csharp
-// Stop running tasks and wait for tasks to complete before returning
-// unless the timeout expires.
-private void Stop(TimeSpan timeout)
-{
-  Trace.TraceInformation("Stop called. Canceling tasks.");
-  // Cancel running tasks.
-  cts.Cancel();
-
-  Trace.TraceInformation("Waiting for canceled tasks to finish and return");
-
-  // Wait for all the tasks to complete before returning. Note that the
-  // emulator currently allows 30 seconds and Azure allows five
-  // minutes for processing to complete.
-  try
-  {
-    Task.WaitAll(tasks.ToArray(), timeout);
-  }
-  catch (AggregateException ex)
-  {
-    Trace.TraceError(ex.Message);
-
-    // If any of the inner exceptions in the aggregate exception
-    // are not cancellation exceptions then rethrow the exception.
-    ex.Handle(innerEx => (innerEx is OperationCanceledException));
-  }
-}
-```
+- Azure App Service and Azure Functions: Deploy shared App Service plans, which represent the hosting server infrastructure. One or more apps can be configured to run on the same computing resources (or in the same App Service plan).
+- Azure Container Apps: Deploy container apps to the same shared environments; especially in situations when you need to manage related services or you need to deploy different applications to the same virtual network.
+- Azure Kubernetes Service (AKS): Deploy shared pods, with a multitenancy-aware application.
```

Comment:
@ckittel  what's your suggestion here ?

"Azure Kubernetes Service (AKS): Deploy shared pods, with a multitenancy-aware application."

Reply:
Azure Kubernetes Service (AKS): AKS is a container-based hosting infrastructure in which multiple applications or application components can be configured to run co-located on the same computing resources (nodes), grouped by computational requirements such as CPU or memory needs (node pools).

What about something like this?

Reply:
Perfect ! Done.